.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,36 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: "[BUG][...]:"
+labels: BUG
+assignees: MChoi-git
+
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**To Reproduce**
```

Comment:
These reproduction steps seem more appropriate for a web app. Can we tailor this for bugs that are likely to arise in our use case?